packages/directlinespeech/src/index.js

```diff
@@ -0,0 +1,14 @@
+/* global process */
+
+import createAdapters from './createAdapters';
+
+export { createAdapters };
+
+if (typeof HTMLDocument !== 'undefined' && typeof document !== 'undefined' && document instanceof HTMLDocument) {
+  const meta = document.createElement('meta');
+
+  meta.setAttribute('name', 'botframework-directlinespeech:version');
+  meta.setAttribute('content', process.env.NPM_PACKAGE_VERSION);
```

Comment:
Can you check the .html files for meta device-width?

Reply:
This meta tag is for identifying the version of DLSpeech SDK using `<meta>` tag.